client/lib/domains/registration/availability-messages.js

```diff
@@ -198,11 +198,16 @@ function getAvailabilityNotice( domain, error, errorData ) {
 		case domainAvailability.TLD_NOT_SUPPORTED:
 		case domainAvailability.TLD_NOT_SUPPORTED_TEMPORARILY:
 		case domainAvailability.UNKNOWN:
-		case domainAvailability.EMPTY_RESULTS:
```

Comment:
Hmm, we should probably test all the possible cases. Don't recall why this was here.

Reply:
At this point, I believe that the only reason we would get empty results is because the query we send to the suggestion engine is empty or invalid after sanitization.

Reply:
I recall now. If both the availability and the suggestions return an error, they quickly switch between the two. :) So we will show the trademark first for availability, then switch to `EMPTY_RESULTS`. Bah, I hate these cases where we handle two data sources and handle them as one.

Reply:
Example:

- Force the suggestion endpoint to return the empty results error
- Incognito window
- /start/domain
- Search for wordpress-ninja.com
- See what happens.